torch/_tensor_str.py

```diff
@@ -527,11 +527,18 @@ def _str_intern(inp, *, tensor_contents=None):
             def indented_str(s, indent):
                 return "\n".join(f"  {line}" for line in s.split("\n"))
 
-            strs = ",\n".join(
-                indented_str(str(t), indent + 1)
-                for t in torch.ops.aten.unbind.int(self, 0)
-            )
-            tensor_str = f"[\n{strs}\n]"
+            if self.is_meta:
+                suffixes.append("size=" + str(tuple(self.shape)))
+                if self.dtype != torch.get_default_dtype():
+                    suffixes.append("dtype=" + str(self.dtype))
+                if not custom_contents_provided:
+                    tensor_str = "..."
+            else:
+                strs = ",\n".join(
+                    indented_str(str(t), indent + 1)
+                    for t in torch.ops.aten.unbind.int(self, 0)
+                )
+                tensor_str = f"[\n{strs}\n]"
```

Comment:
leftover from old approach; this isn't called if we use python `NestedTensor` exclusively for meta / fake NTs. Rather, `NestedTensor.__repr__()` will be called.